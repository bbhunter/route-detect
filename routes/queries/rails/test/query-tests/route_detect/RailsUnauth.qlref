route_detect/RailsUnauth.ql
