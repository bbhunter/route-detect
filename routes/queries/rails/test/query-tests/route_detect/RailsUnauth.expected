| app/config/routes.rb:3:3:3:41 | call to resources | Found unauthed Rails route, method=get path=users controller=users action=index |
| app/config/routes.rb:3:3:3:41 | call to resources | Found unauthed Rails route, method=get path=users/:id controller=users action=show |
| app/config/routes.rb:4:3:4:41 | call to resources | Found unauthed Rails route, method=get path=posts controller=posts action=index |
